src/components/tool/guide/Config.js

```diff
@@ -86,9 +89,23 @@ export const getCpuModel = (config, onChange) => () => {
 
   let alertMsg;
   if (config?.cpuModel === "E7-8890v4") {
-    alertMsg = "The E7-8890v4 CPU should only be used for large memory jobs.";
+    alertMsg =
+      "The E7-8890v4 CPU should only be used for jobs with more than 512 GB of ram.";
```

Comment:
Just a thought: replacing `with` by `require` sounds more appropriate? Also should `ram` be capitals? It's abbreviation sames as `CPU`.